### ISSUE TITLE
Unable to run Raku REPL or scripts

### ISSUE THREAD

**Issue by ArthurZ** [rakudo/rakudo#3956]
## The Problem
Unable to run Raku REPL or scripts
## Expected Behavior
Able to run Raku REPL or scripts
## Actual Behavior
Error
`Unhandled exception: Missing or wrong version of dependency 'C:\rakudo\bin\..\share\nqp\lib\MAST\Nodes.nqp' (from 'src/Perl6/Pod.nqp')`
## Steps to Reproduce

Install Raku either via `choco install rakudostar`, or by downloading *rakudo-moar-2020.09-01-win-x86_64.zip*
## Environment
* Operating system: Win 10 PRO
* Compiler version (`perl6 -v` or `raku -v`): Unable to retrieve

**Comment by patrickbkr** [comment]
Thanks for the report!
Can you give more details on how you try to run Raku?
What folder did you extract / install it into?
How did you call it? Powershell? CMD? lnk file?
Can you check if any Raku specific environment variables are set? (anything with "NQP", "RAKUDO" or "PERL6" in it)
Are there any previous installations present on the system?

**Comment by ArthurZ** [comment]
Thank you @patrickbkr for stepping in to help.
I had an earlier version perhaps and installed the newer on top.
I download the 64 Bit ver. from https://rakudo.org/downloads and got the error.
I then deleted the folder, installed using Chocolatey - same result.
Tried Rakubrew to no avail. It simply kept telling to init, did not work in shim mode, and my patience exploded after 
`»  ./rakubrew download                                              
Couldn't find a precomp release for OS: "win", architecture: "x86_64"`
Rebooted, re-extracted rakudo-moar-2020.09-01-win-x86_64.zip to the same directory
`C:\Rakudo` and voila - I got the Raku REPL :-)
